Detectors/ITSMFT/ITS/base/include/ITSBase/GeometryTGeo.h

```diff
@@ -103,10 +103,12 @@ class GeometryTGeo : public o2::itsmft::GeometryTGeo
   int getNumberOfChipsPerModule(int lay) const { return mNumberOfChipsPerModule[lay]; }
   int getNumberOfChipsPerHalfStave(int lay) const { return mNumberOfChipsPerHalfStave[lay]; }
   int getNumberOfChipsPerStave(int lay) const { return mNumberOfChipsPerStave[lay]; }
+  int getNumberOfChipsPerHalfBarrel(int lay) const { return mNumberOfChipsPerHalfBarrel[lay]; }
   int getNumberOfChipsPerLayer(int lay) const { return mNumberOfChipsPerLayer[lay]; }
   int getNumberOfModules(int lay) const { return mNumberOfModules[lay]; }
   int getNumberOfHalfStaves(int lay) const { return mNumberOfHalfStaves[lay]; }
   int getNumberOfStaves(int lay) const { return mNumberOfStaves[lay]; }
+  int getNumberOfHalfBarrels(int lay) const { return mNumberOfHalfBarrels[lay]; }
```

Comment:
@mario6829 Do we need this function and the corresponding std::vector ?  The number of half-barrels is always 2, is not it ?

Reply:
Hi @iouribelikov I just wrote it like the other ones (pretty like the number of layers, which is actually fixed too but it has a member and a getter), but yes, I can change it into a constant (or in a single-valued member instead of a vector).  @shahor02 do you agree ?

Reply:
Sure

Reply:
Done: I changed it to a single-valued member, the same for all layers.